spec/controllers/category_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe CategoryController, type: :controller do
+  describe "GET #index" do
+    it "returns http success" do
+      get :index
+      expect(response).to have_http_status(301)
+    end
+  end
+
+  # describe "GET #show" do
```

Comment:
is this meant to be commented out?

Reply:
This test has been removed, since I agree that we need to improve the test set up since so many tests now rely on a common set up (as in your other comment).